strawberryfields/engine.py

```diff
@@ -540,17 +548,19 @@ def run(self, program: Program, *, compile_options=None, **kwargs) -> Optional[R
             ``None`` otherwise
         """
         job = self.run_async(program, compile_options=compile_options, **kwargs)
+        log.info("Submitted job {}".format(job.id))
```

Comment:
maybe rather supplement the id here: https://github.com/XanaduAI/strawberryfields/blob/f6ac22b00c8a085e0fbd75d3a19b631e634e4c0f/strawberryfields/api/connection.py#L158
to avoid logging something similar twice, and to also get that log message for lower-level job creation routines?

Reply:
or is it desired to only log this for blocking job requests?

Reply:
That's a good one, thanks @lneuhaus! I think it would be nice to have it for both blocking and non-blocking, it was just doable to get it for now with non-blocking.